web/impact/impact/v1/helpers/mentor_program_office_hour_helper.py

```diff
@@ -19,9 +19,12 @@
     "end_date_time": OPTIONAL_STRING_FIELD,
     "mentor_id": OPTIONAL_INTEGER_FIELD,
     "mentor_name": OPTIONAL_STRING_FIELD,
+    "mentor_title": OPTIONAL_STRING_FIELD,
```

Comment:
I'm not seeing how changes in this file are related to this ticket

Reply:
Nice catch